snafu/log_generator_wrapper/log_generator_wrapper.py

```diff
@@ -59,6 +59,7 @@ def __init__(self, parent_parser):
         )
         parser.add_argument("--kafka-bootstrap-server", help="The Kafka Service to connect to")
         parser.add_argument("--kafka-topic", help="The Kafka topic to verify logs")
+        parser.add_argument("--kafka-check", help="Verify if logs made it to kafka sink", action="store_true")
```

Comment:
@smalleni Curious why you are adding a verification true|false. If you are providing the topic/bootstrap then the verification would be implied. Or is there another time/reason you would pass that info but not want to verify?

Reply:
That's fair.  I just wanted to confirm the intent really.